lib/DDG/Spice/GithubIssues.pm

```diff
@@ -0,0 +1,44 @@
+package DDG::Spice::GithubIssues;
+# ABSTRACT: Search Github for Issues related to the query
+
+use DDG::Spice;
+use strict;
+use warnings;
+use Text::Trim;
+
+spice is_cached => 1;
+spice proxy_cache_valid => '200 1h';
+
+spice wrap_jsonp_callback => 0;
+
+spice to => 'https://api.github.com/search/issues?q=$1&callback={{callback}}';
+
+my @languages = share("languages.txt")->slurp;
+@languages = sort { length($b) <=> length($a) } @languages;
+
+chomp(@languages);
+my $langs = join("|", map(quotemeta, @languages));
+
+triggers startend => 'github issue';
```

Comment:
I think we need to make this an `end` trigger only. Looking at some queries, none of the ones that start with `github issue` seem to be a search for specific issues